apis/imdb.py

```diff
@@ -43,7 +49,7 @@ def fetch_imdb(url: str) -> Dict:
         
         api_data["trailer"] = Trailer
 
-        wiki_url = parse_wiki_url(title)
+        wiki_url = parse_wiki_url(title, 'movie')

```

Comment:
nit: you should make the 2nd params enums
`wiki_url = parse_wiki_url(title, WikiUrlTitle.movie)`

Reply:
Cool! That would be much better.